src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="get"
+    data-qa="big"
+    >
+      <label class="searchFormBig">
+        <img
+        src="images/Search.svg"
+        alt="searchIcon"
+        class="imageBig"
+        >
+        <input
+        class="inputBig"
+        type="text"
+        name="bigSearchBar"
+        data-qa="keypress"
+        placeholder="Try &#34Los Angeles&#34"
+        >
```

Comment:
[Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md), the 5th item